Mathlib/Analysis/Calculus/FDeriv/Mul.lean

```diff
@@ -22,7 +22,7 @@ This file contains the usual formulas (and existence assertions) for the derivat
 
 
 open scoped Classical
-open Filter Asymptotics ContinuousLinearMap Set Metric Topology NNReal ENNReal
+open Filter Asymptotics ContinuousLinearMap Set Metric Topology NNReal ENNReal BigOperators
```

Comment:
git glitch. It's always a good idea to check your PR's on github to make sure you're not modifying any files you didn't mean to.